segtypes/common/rodata.py

```diff
@@ -60,6 +60,10 @@ def split(self, rom_bytes: bytes):
 
                     path = path_folder / f"{rodataSym.getName()}.s"
                     with open(path, "w", newline="\n") as f:
-                        f.write('.include "macro.inc"\n\n')
-                        f.write(".section .rodata\n\n")
+                        if options.compiler != compiler.SN64:
```

Comment:
I think instead of this if statement, we should just have a compiler property that is different for SN64